scripts/genproto.sh

```diff
@@ -95,18 +95,34 @@ run_go_tool github.com/hexfusion/schwag -input=Documentation/dev-guide/apispec/s
 if [ "$1" != "--skip-protodoc" ]; then
   log_callout "protodoc is auto-generating grpc API reference documentation..."
 
+  # API reference
   run rm -rf Documentation/dev-guide/api_reference_v3.md
```

Comment:
Once we know for sure if/where we're going to move these files, I'd like to set an env var for them as we use them a couple times.